blog_post.md

```diff
@@ -140,18 +140,14 @@ where the last equality is a general result for geometric series. This means tha
     - reward clipping -->
 
 ## Experimental setup
+We try to follow the experimental setup from the [DQN paper](https://www.cs.toronto.edu/~vmnih/docs/dqn.pdf) closely. The loss term we use is the mean-squared error which is clipped in the range [-1; 1], as it was reported to improve the stability of the algorithm. The error is optimized by [Adam](https://arxiv.org/pdf/1412.6980.pdf) with a learning rate $$\alpha = 0.001$$. The choice of optimizer deviates from the original paper but has shown great success in deep learning recently. Additional experiments with different values of the learning rate and the contribution of error clipping are left for future work.
```

Comment:
We need to add that we don't do clamping for cartpole as it actually prevents learning